### PR TITLE
Implement todo

### PR DESCRIPTION
**Pull request by anlithov**
Live Todo list. Pages. Add todo module

### CODE REVIEW

File: mobile/src/screens/TodoStack/components/forms/TodoElement.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+interface TodoElementProps {
+  item: IServerTodo
+}
+
+
+const TodoElement: FC<TodoElementProps> = (props) => {
+  const { mutate } = useMutateActions(TodoActionTypes.DELETE_TODO);
+  const navigation = useNavigation<any>();
+  const goToDelete = () => {
+    console.log('Start Mutate');
```
Comment (CodeGeneration-2020) [inline mobile/src/screens/TodoStack/components/forms/TodoElement.tsx:18]: Remove logs

File: mobile/src/screens/TodoStack/components/forms/TodoElement.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+import { useNavigation } from '@react-navigation/native';
+
+interface TodoElementProps {
+  item: IServerTodo
+}
+
+
+const TodoElement: FC<TodoElementProps> = (props) => {
+  const { mutate } = useMutateActions(TodoActionTypes.DELETE_TODO);
+  const navigation = useNavigation<any>();
```
Comment (CodeGeneration-2020) [inline mobile/src/screens/TodoStack/components/forms/TodoElement.tsx:16]: any

File: mobile/src/screens/TodoStack/components/forms/TodoElement.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
+import React, { FC } from 'react';
+import styled from 'styled-components/native';
+import { TouchableOpacity } from 'react-native';
+
+import { IServerTodo, TodoActionTypes, TodoScreenTypes } from '../../../../types/todo.type';
+import { useMutateActions } from '../../../../hooks/useMutateActions';
+import { useNavigation } from '@react-navigation/native';
+
+interface TodoElementProps {
```
Comment (CodeGeneration-2020) [inline mobile/src/screens/TodoStack/components/forms/TodoElement.tsx:9]: Interfaces should start with I letter

File: mobile/src/service/todo.service.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import { IServerTodo, ITodo } from '../types/todo.type';
+
+
+export class TodoService {
```
Comment (CodeGeneration-2020) [inline mobile/src/service/todo.service.ts:5]: Create base service example of it you can find in our styleguide

**Review by CodeGeneration-2020 — CHANGES_REQUESTED** [review]

File: mobile/src/service/todo.service.ts (hunk 4 above)
Comment (anlithov) [reply mobile/src/service/todo.service.ts:5]: Don't understand the issue. Tried to make service according to styles with some minor changes.
What do I need to fix exactly?

Comment (anlithov) [reply mobile/src/service/todo.service.ts]: Done. Found guide. Implemented http service and rebuilt of todo service in last commit

**Pull request merged**